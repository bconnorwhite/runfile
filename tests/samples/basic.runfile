# ----------
# Basic Commands
# ----------

# Simple command with no args
hello:
  echo "Hello, World!"

# Command with optional argument
greet name?:
  echo "Hello, ${name:-World}!"

# Command with flags
build --debug --release --output=<file>:
  echo "Building with debug: $DEBUG"
  echo "Release mode: $RELEASE"
  echo "Output: $OUTPUT"
