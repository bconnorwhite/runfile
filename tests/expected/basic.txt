Basic Commands
  hello         # Simple command with no args
  greet         # Command with optional argument
    name?
  build         # Command with flags
    --debug
    --release
    --output

